docs/siem/detections/alerts-ui-manage.asciidoc

```diff
@@ -0,0 +1,98 @@
+[[alerts-ui-manage]]
+[role="xpack"]
+== Managing detection alerts
+
+The Detections page displays all alerts. From the Alerts table, you can change
+an alert's status, and start investigating and analyzing alerts in Timeline.
+
+TIP: From Timeline, you can <<cases-ui-open, create cases>> to track issues and
+share information with colleagues.
+
+To view detection alerts created by a specific rule, you can:
+
+* Filter for a specific rule in the KQL bar (for example,
+`signal.rule.name :"SSH (Secure Shell) from the Internet"`).
+* View detection alerts in the *Rule details* page (click
+*Manage detection rules* -> rule name in the *All rules* table).
+
+NOTE: KQL autocomplete for `.siem-signals-*` indices is available on the 
+*Detections* and *Rule details* pages, and in Timeline when either `All` or
+`Detection alerts` is selected. 
+
+TIP: Use the icons in the upper left corner of the Alerts table to customize
+displayed columns and row renderers, and view the table in full screen mode.
```

Comment:
Also of note is that these configurations (along with column order and alerts per page) will now persist to the user's local storage. There is separate persistence for the main Detections page Alerts Table and the Rule Details Alerts Table to allow greater flexibility. This was added via https://github.com/elastic/kibana/pull/67156/ thanks to @cnasikas! 🙂

Reply:
Great, but let's add this to the new What's New section.